help/destinations/catalog/analytics/gainsight-px.md

```diff
@@ -0,0 +1,107 @@
+
+### Authenticate to destination {#authenticate}
+
+To authenticate to the destination, fill in the required fields and select **[!UICONTROL Connect to destination]**.
+
+![Authentication screenshot](../../assets/catalog/analytics/gainsight_px/auth-screen.png)
+* **[!UICONTROL Password]**: The password used to login to the [Gainsight PX UI](https://app.aptrinsic.com)
+* **[!UICONTROL Client ID]**: The Gainsight PX subscription ID on the [Company Details page](https://app.aptrinsic.com/settings/subscription)
+* **[!UICONTROL Client secret]**: The OAuth secret generated on bottom of the [Company Details page](https://app.aptrinsic.com/settings/subscription)
+* **[!UICONTROL Username]**: The email used to login to the [Gainsight PX UI](https://app.aptrinsic.com)
+
+### Fill in destination details {#destination-details}
+
+To configure details for the destination, fill in the required and optional fields below. An asterisk next to a field in the UI indicates that the field is required.
+
+![destination details screenshot](../../assets/catalog/analytics/gainsight_px/destination_details.png)
+
+*  **[!UICONTROL Name]**: A name by which you will recognize this destination in the future.
+*  **[!UICONTROL Description]**: A description that will help you identify this destination in the future.
+
+When you are finished providing details for your destination connection, select **[!UICONTROL Next]**.
+
+## Activate segments to this destination {#activate}
+
+>[!IMPORTANT]
+> 
+>* To activate data, you need the **[!UICONTROL Manage Destinations]**, **[!UICONTROL Activate Destinations]**, **[!UICONTROL View Profiles]**, and **[!UICONTROL View Segments]** [access control permissions](/help/access-control/home.md#permissions). Read the [access control overview](/help/access-control/ui/overview.md) or contact your product administrator to obtain the required permissions.
+>* To export *identities*, you need the **[!UICONTROL View Identity Graph]** [access control permission](/help/access-control/home.md#permissions). <br> ![Select identity namespace highlighted in the workflow to activate audiences to destinations.](/help/destinations/assets/overview/export-identities-to-destination.png "Select identity namespace highlighted in the workflow to activate audiences to destinations."){width="100" zoomable="yes"}
+
+Read [Activate profiles and segments to streaming segment export destinations](/help/destinations/ui/activate-segment-streaming-destinations.md) for instructions on activating audience segments to this destination.
+
+### Map attributes and identities {#map}
+
+The only relevant mapping is between the Adobe identity and the Gainsight PX identity.  Choose the appropriate identity source field on the Adobe side and map it to the IDENTIFY_ID (the only choice) field on the target side.  It is important that the source field have the exact same value that is passed during the Gainsight PX identify step.
```

Comment:
This section needs to be rephrased in a clearer and more user-centric manner, but before we do that, I need to understand the functionality that you are describing here.

- Are there are constraints around the supported source identities? Your screenshot shows the ECID as a source identity. Is that the only one supported?

Also, I don't understand the last sentence: _It is important that the source field have the exact same value that is passed during the Gainsight PX identify step_.

Please provide some more details about the supported identities and how the mapping is supposed to be done here, since it all sounds very confusing at this point. We'll work together on a rephrasing once I understand what the meaning should be.